docs/environment.yml

```diff
@@ -7,7 +7,7 @@ dependencies:
 - nbformat
 - jupyter_client
 - ipython
-- sphinx>=1.3.6
+- sphinx>=1.3.6,!=1.5.0,!=1.5.1
```

Comment:
OK. Let's remember to keep an eye on it when Sphinx 1.5.2 comes out to see if it breaks again.

I've just spotted that RTD has email notifications for failures. I'll sign myself up for that,